src/scss/_mixins.scss

```diff
@@ -34,21 +34,12 @@
 /// @param {String|list} $propertyList [all]
 @mixin oColorsFor($useCaseList, $propertyList: all) {
 	@if ($propertyList == 'all' or index($propertyList, 'background')) {
-		$col: oColorsGetColorFor($useCaseList, background, (default:undefined));
-		@if ($col != undefined) {
-			background-color: $col;
-		}
+		background-color: oColorsGetColorFor($useCaseList, background, null);
```

Comment:
It won't output anything at all (see PR description). This is a useful Sass thing: `prop: null` compiles to nothing.